code/modules/economy/traders/gragg.dm

```diff
@@ -155,5 +155,5 @@
 /datum/commodity/trader/gragg/starstone
 	comname = "Rare star-shaped jewel"
 	comtype = /obj/item/raw_material/starstone
-	price_boundary = list(3000000,3000000)
+	price_boundary = list(500000,4500000)
```

Comment:
Dont think this nerf is fair.

Reply:
I don't think this is a flat nerf, since even $500,000 is a very big sum for cargo. I was just personally tired of it being a static value that never changed. Now it can be higher than the original, or it can have low points. I think it's overall a good change since it'll encourage looking for other options if it's not particularly high that shift. Plus this PR is aiming at making more items worth it to sell, like ores, herbs, etc.